ReactSkia/sdk/NotificationCenter.cpp

```diff
@@ -26,6 +26,7 @@
 #include "NotificationCenter.h"
 
 static std::unique_ptr<NotificationCenter> defaultCenter_;
+static std::unique_ptr<NotificationCenter> OSKCenter_;//OnScreenKeyboard Notification center
```

Comment:
Just a thought,  could be part of improvement as well.
NotificationCenter in general does not know about OSK .
We may need to have a mechanism where NotificationCenter needs to provide interface for clients to register . Here OSK is one of the client. In future,we may have multiple clients.